helpers/config.py

```diff
@@ -811,14 +812,20 @@ def __questions_backup(self):
 
                     if self.backend_questions:
                         
-                        if self.primary_backend:
-                            CLI.colored_print("Run PostgreSQL backup from primary backend server?",
-                                            CLI.COLOR_SUCCESS)
-                            CLI.colored_print("\t1) Yes")
-                            CLI.colored_print("\t2) No")
-                            self.__config["backup_from_primary"] = CLI.get_response(
-                                [Config.TRUE, Config.FALSE],
-                                self.__config.get("backup_from_primary", Config.TRUE))
+                        if self.__config['use_wal_e'] == Config.TRUE:
+                            self.__config["backup_from_primary"] = Config.FALSE
```

Comment:
You should not have copied my mistakes :P 😇 

Backup is enforce on primary with WAL-E 

```
self.__config["backup_from_primary"] = Config.TRUE 
```